images/expansionhunter/Dockerfile

```diff
@@ -0,0 +1,50 @@
+
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG CONDA_RELEASE="4.6.14"
+ARG CONDA_INSTALL_DIR="/opt/conda"
+ARG CONDA_BIN=${CONDA_INSTALL_DIR}/bin
+ARG CONDA_CMD=${CONDA_BIN}/conda
+ENV PATH=${CONDA_BIN}:$PATH
+ENV EH_VERSION=v5.0.0
+
+RUN apt-get update && apt-get install --no-install-recommends -qqy \
+    python3-dev \
+    python3-pip \
+    python \
+    python-dev \
+    awscli \
+    build-essential \
+    git \
+    libbz2-dev \
+    liblzma-dev \
+    make \
+    pkg-config \
+    wget \
+    unzip \
+    zlib1g-dev \
+    libcurl4-openssl-dev \
+    libssl-dev
+
+# Install and configure conda
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_RELEASE}-Linux-x86_64.sh -O /tmp/miniconda.sh && \
```

Comment:
Is there any specific reason for installing conda? The packages you install below are from `pip`, and could probably save a bit of space by not including these.

Reply:
No, I thought there was some relationship between pip and conda. I have removed conda.